lib/cli/src/commands/package/common/macros.rs

```diff
@@ -0,0 +1,79 @@
+macro_rules! make_pb {
+    ($self:ident, $msg:expr) => {{
+        let pb = indicatif::ProgressBar::new_spinner();
+        if $self.quiet {
+            pb.set_draw_target(indicatif::ProgressDrawTarget::hidden());
+        }
+
+        pb.enable_steady_tick(std::time::Duration::from_millis(500));
+        pb.set_style(
+            indicatif::ProgressStyle::with_template("{spinner:.magenta} {msg}")
+                .unwrap()
+                .tick_strings(&["✶", "✸", "✹", "✺", "✹", "✷"]),
+        );
+
+        pb.set_message($msg);
+        pb
+    }};
+
+    ($self:ident, $msg:expr, $($spinner:expr),+) => {{
+        let pb = indicatif::ProgressBar::new_spinner();
+        if $self.quiet {
```

Comment:
Instead of passing self, we should just pass `quiet`. That way it can be reused in places where there is no self.

```suggestion
    ($quiet:ident, $msg:expr, $($spinner:expr),+) => {{
        let pb = indicatif::ProgressBar::new_spinner();
        if $quiet {
```

Reply:
https://github.com/wasmerio/wasmer/pull/4643/commits/64e969053f7124302e86fbb97c431abd86f99fb2#diff-171b04fbf128026d96a51378fbf46e7b2609e4aa69a86858705d394bb988fcb7R2